docs/notebook/Tutorial.ipynb

```diff
@@ -731,7 +731,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can add additional interpolation types using custom resolvers. This example creates a resolver that adds 10 the the given value."
+    "You can add additional interpolation types using custom resolvers, which take strings as inputs. This example creates a resolver that adds 10 to the given value (note the need to cast it to `int`)."
```

Comment:
I will improve this when I have a proper grammar. for now this is indeed taking strings which is unfortunate.

Reply:
Cool, I just added this comment because it's easy to do something like `${plus:1,2}` and end up with `"12"` instead of `3` :)